Src/SarifPatternMatcher/Validation.cs

```diff
@@ -5,9 +5,46 @@ namespace Microsoft.CodeAnalysis.SarifPatternMatcher
 {
     public enum Validation
     {
+        /// <summary>
+        /// A default value that reflects an uninitialized state,
+        /// (i.e., no useful validation state is available).
+        /// </summary>
         None = 0,
+
+        /// <summary>
+        /// A validator for the current matched pattern
+        /// could not be found.
+        /// </summary>
         ValidatorNotFound = None,
```

Comment:
>None [](start = 28, length = 4)

If we set None, would this be able to distinguish a value 0 from `None` or `ValidatorNotFound`?

Reply:
Yes, so None is basically an entirely uninitialized value. 

---
In reply to: [541451683](https://github.com/microsoft/sarif-pattern-matcher/pull/21#discussion_r541451683) [](ancestors = 541451683)